src/main/java/org/jitsi/videobridge/cc/AdaptiveTrackProjection.java

```diff
@@ -215,6 +215,19 @@ public boolean accept(@NotNull VideoRtpPacket videoRtpPacket)
             logger.warn(
                 "Dropping an RTP packet, no quality index (SSRC=" +
                     videoRtpPacket.getSsrc() + ").");
+            MediaStreamTrackDesc sourceTrack = getSource();
+            if (sourceTrack != null)
+            {
+                logger.warn(
```

Comment:
confused by this, if we find the source track is not null we drop it?  and if it is null we  drop it?

Reply:
oh, ok...after expanding i think i see now. nevermind.